app/src/main/java/com/cornellappdev/android/eatery/CustomPager.java

```diff
@@ -6,38 +6,39 @@
 import android.view.View;
 
 public class CustomPager extends ViewPager {
-  private View mCurrentView;
+    private View mCurrentView;
 
-  public CustomPager(Context context) {
-    super(context);
-  }
-
-  public CustomPager(Context context, AttributeSet attrs) {
-    super(context, attrs);
-  }
+    public CustomPager(Context context) {
+        super(context);
+    }
 
-  /**
-   * Note(lesley): Code is either from StackOverflow or official docs, allows for ViewPager to be
-   * visible inside of MenuActivity. TBH not sure how it works, but it has something to do with
-   * resizing the Viewpager window
-   */
-  @Override
-  public void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
-    if (mCurrentView == null) {
-      super.onMeasure(widthMeasureSpec, heightMeasureSpec);
-      return;
+    public CustomPager(Context context, AttributeSet attrs) {
+        super(context, attrs);
     }
-    int height = 0;
-    mCurrentView.measure(widthMeasureSpec, MeasureSpec.makeMeasureSpec(0, MeasureSpec.UNSPECIFIED));
-    int h = mCurrentView.getMeasuredHeight();
-    if (h > height) height = h;
-    heightMeasureSpec = MeasureSpec.makeMeasureSpec(height, MeasureSpec.EXACTLY);
 
-    super.onMeasure(widthMeasureSpec, heightMeasureSpec);
-  }
+    /**
+     * Note(lesley): Code is either from StackOverflow or official docs, allows for ViewPager to be
```

Comment:
If we use uncommented code that's not ours, can we link to the docs/stackoverflow post?